packages/components/calendar/src/multi-date-picker.tsx

```diff
@@ -182,6 +183,10 @@ export const MultiDatePicker = forwardRef<MultiDatePickerProps, "input">(
 
             <Portal {...portalProps}>
               <PopoverContent
+                id={id}
+                role="dialog"
+                aria-modal="true"
+                aria-multiselectable="true"
```

Comment:
I think `aria-multiselectable` needs to be set to the `grid`role.

Reply:
source: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable#used_with_aria-selected

Reply:
@108yen  @mokun632 
Thank you for comment!
I fixed now. please re-review!

Reply:
@cidkumagai 
It is not passed to the `grid`role component.

Reply:
@108yen 
Sorry. Please review again as we revise.